meinberlin/apps/budgeting/views.py

```diff
@@ -20,6 +20,8 @@ def get_ordering_choices(view):
     choices = (('-created', _('Most recent')),)
     if view.module.has_feature('rate', models.Proposal):
         choices += ('-positive_rating_count', _('Most popular')),
+    elif view.module.has_feature('support', models.Proposal):
+        choices += ('-positive_rating_count', _('Most support')),
```

Comment:
This is for the list, we will not show later, right? But good to have it here, analogous to the rating!

Reply:
No, it is for the pop up to work, because if the most support filter is set in list and it doesnt know it here, it breaks..